tests/kani/DynTrait/boxed_closure_fail.rs

```diff
@@ -1,15 +0,0 @@
-// Copyright Kani Contributors
-// SPDX-License-Identifier: Apache-2.0 OR MIT
-// kani-verify-fail
-
-// Check that we can codegen a boxed dyn closure and fail an inner assertion
-#[kani::proof]
-fn main() {
-    // Create a boxed once-callable closure
-    let f: Box<dyn FnOnce(i32)> = Box::new(|x| {
-        kani::expect_fail(x == 2, "Wrong int");
```

Comment:
What happened to this test? Did you forget to do a git add on the new ones??

Reply:
I deliberately deleted most of the `*_fail` tests because they are useless. For example, there is a corresponding test called `boxed_closure.rs` which is almost identical to this one except that it uses asserts instead of `expect_fail`:

https://github.com/model-checking/kani/blob/main/tests/kani/DynTrait/boxed_closure.rs

This test is passes, and thus subsumes the `boxed_closure_fail.rs` test. This is the same case for the tests I deleted (I checked them one by one).

See the call-outs for more details.

Reply:
Ok. Sorry about that